<map id="openage::error::Error::trim_backtrace" name="openage::error::Error::trim_backtrace">
<area shape="rect" id="node2" href="$db/dbd/namespaceopenage_1_1error.xhtml#a036babcdd58261ba26288431876e7b02" title="openage::error::demo" alt="" coords="195,5,340,32"/>
<area shape="rect" id="node3" href="$d0/df4/classopenage_1_1error_1_1_error.xhtml#a27cbe56bc7dbab4d399072ef87fe2014" title="Stores a pointer to the currently&#45;handled exception in this&#45;&gt;cause. " alt="" coords="197,57,338,98"/>
<area shape="rect" id="node4" href="$d0/df4/classopenage_1_1error_1_1_error.xhtml#a755a7891dfe08412bc2312f29b0e618d" title="openage::error::Error\l::Error" alt="" coords="405,23,546,65"/>
<area shape="rect" id="node5" href="$d9/dde/namespaceopenage_1_1pyinterface.xhtml#a89529648faa2ef530fefcc90f1c1ab70" title="Analyzes the current Python exception, and creates and throws an analogous C++ PyException object..." alt="" coords="388,89,563,130"/>
<area shape="rect" id="node6" href="$d0/d44/classopenage_1_1pyinterface_1_1_func.xhtml#acea6cb67bc0e8ea78d513f6db4e2e477" title="Specialization for bind() with void return types. " alt="" coords="611,81,813,137"/>
</map>
